app/models/appointment.rb

```diff
@@ -1,14 +1,16 @@
 class Appointment < ActiveRecord::Base
+  COMPLETE = 'Complete'.freeze
```

Comment:
You might argue that replacing a literal (that's used once) with a constant of the same name is maybe overkill?